apidoc/Titanium/UI/TextArea.yml

```diff
@@ -296,6 +296,16 @@ properties:
     constants: Titanium.UI.KEYBOARD_*
     default: <Titanium.UI.KEYBOARD_DEFAULT>
     platforms: [android, iphone, ipad]
+
+  - name: textContentType
+    summary: Provide the keyboard with extra information about the semantic intent of the text 
+        document.
+    type: String
+    constants: Titanium.UI.iOS.CONTENT_TYPE_*
```

Comment:
I think if the method accepts single constants it should be mentioned like   Titanium.UI.iOS.CONTENT_TYPE_* rather [Titanium.UI.iOS.CONTENT_TYPE_*] . e.g. see "keyboardType" inside TextArea.yml.